generators/ci-cd/templates/circle.yml.ejs

```diff
@@ -103,23 +95,23 @@ jobs:
                 <%_ } _%>
             <%_ } else if (buildTool === 'gradle') { _%>
                 <%_ if (cicdIntegrations.includes('heroku')) { _%>
-            -run:
+            - run:
                 name: Deploy to Heroku
                 command: './gradlew deployHeroku --no-daemon'
                 <%_ } _%>
             <%_ } _%>
             <%_ if (testFrameworks.includes('cypress') || testFrameworks.includes('protractor')) {
-                const commented = applicationType === 'gateway' ? '# ' : ''; _%>
-            <%= commented %>-run:
+                const commented = '# '; _%>
```

Comment:
@pascalgrimaud : Just a bit confused with this commented out section. Why do we have this e2e section if it doesn't work with CircleCI? Would we just remove it? 🤔

Reply:
to make it works, the user needs to remove the 127.0.0.1 part in docker-compose files, and as the change is easy, I thought it's a good idea to let this part.
But if you think we should remove it, I can too

Reply:
Ah, I get it. No, it's fine I think; makes sense. 👍🏽